<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>res</class>
 <widget class="QWidget" name="res">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ResText">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Success</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ChoseList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>256</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Shower">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Confirm">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pic">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pic_y">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
